webapp/src/views/projects/languages/ProjectSettingsLanguages.tsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import { Box, Typography } from '@material-ui/core';
+import { T, useTranslate } from '@tolgee/react';
+import { useQueryClient } from 'react-query';
+import { Link } from 'react-router-dom';
+
+import { SettingsIconButton } from 'tg.component/common/buttons/SettingsIconButton';
+import { CreateSingleLanguage } from 'tg.component/languages/CreateSingleLanguage';
+import { LINKS, PARAMS } from 'tg.constants/links';
+import { useProject } from 'tg.hooks/useProject';
+import { invalidateUrlPrefix, useApiQuery } from 'tg.service/http/useQueryApi';
+import { useGlobalLoading } from 'tg.component/GlobalLoading';
+import { MachineTranslation } from './MachineTranslation/MachineTranslation';
+import { LanguageItem } from './LanguageItem';
+import { useTableStyles } from './tableStyles';
+import clsx from 'clsx';
+import { AutoTranslations } from './AutoTranslations/AutoTranslations';
+
+export const ProjectSettingsLanguages = () => {
+  const tableClasses = useTableStyles();
+  const queryClient = useQueryClient();
+  const project = useProject();
+  const t = useTranslate();
+
+  const languagesLoadable = useApiQuery({
+    url: '/v2/projects/{projectId}/languages',
+    method: 'get',
+    path: { projectId: project.id },
+    query: {
+      size: 1000,
+    },
+  });
+
+  const gridTemplateColumns = '1fr auto auto';
```

Comment:
I would name constant like this with UPPERCASE_SNAKE_CASE

Reply:
Hmm, it's not really that kind of constant, however it's not even necessary to have it in variable